__tests__/comments.tests.js

```diff
@@ -0,0 +1,71 @@
+const app = require('../app');
+const request = require('supertest');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index');
+const sorted = require('jest-sorted')
+
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+afterAll(() => {
+    return db.end()
+})
+
+describe('/api/articles/:article_id/comments', () => {
+    it('404: article not found', async () => {
+        try {
+            const result = await request(app)
+            .get('/api/articles/34234234/comments')
+        } catch(error) {
+            expect(error).toMatchObject({
+                status: 404, 
+                msg: "No article found"
+            })
+        }
+    });
+    it('400: not a number', async () => {
+        try{
+            const { body } = await request(app)
+            .get('/api/articles/e/comments')
+        } catch(error) {
+            expect(error).toMatchObject({
+                status: 400,
+                msg: "Wrong data type"
+            })
+        }
+    });
```

Comment:
Its interesting to see error handling done first, this is generally something we can work on after the main functionality of the endpoint has been completed.